### DISCUSSION TITLE
[Prettier Plugin] Tailwind prettier does not work with pnpm

### DISCUSSION THREAD

**Discussion by LazyClicks** [tailwindlabs/tailwindcss#11625] [Help]
I have a project with qwik and pnpm, I followed all instructions when installing tailwind prettier but it doesn't work, prettier on its own works but the tailwind prettier plugin doesn't
I also tried adding the require into tailwind config but that still didn't help

**Comment by roma-marshall** [comment]
hello @LazyClicks, maybe it's just doesn't load automatically. Check it out: [#4700](https://github.com/pnpm/pnpm/issues/4700) and [#8474](https://github.com/prettier/prettier/issues/8474)

**Comment by Gravy59** [comment]
Try doing `plugins: [require.resolve("prettier-plugin-tailwindcss")]`
